treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
I think it is ok to run this every time we deploy- I would really like upon startup that we automatically fill the data, not wait for a cron job to run.